makeRelease.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
 if [ "$#" -ne 2 ]; then
-    echo "Usage:\n  ./makeRelease 1.2.3rc0 Directory/"
-    exit 1
+  printf "Usage:\n  ./makeRelease release Directory/"
```

Comment:
We use 4 spaces in Java, so I would prefer to keep it this way. I think the old release number example is easier to understand than the word "release".

Reply:
I'll change it to versionNumber, since the version is still old and may be potentially confusing.

Reply:
I would prefer versionName because it is more clear that it does not mean the integer version number. Anyway, I still find the example easier to understand than a description, to be honest.

Reply:
Alright, I'll change it back, then.